packages/taquito/src/contract/interface.ts

```diff
@@ -283,4 +284,34 @@ export interface ContractProvider extends StorageProvider {
    * @param params FailingNoop operation parameter
    */
   failingNoop(params: FailingNoopParams): Promise<FailingNoopOperation>;
+
+  /**
+   *
+   * @description Stake founds. Will sign and inject an operation using the current context
```

Comment:
octez-client desc: `Stake the given amount for the source. The source must be a delegator to be allowed to stake.`